components/suggest/src/pocket_confidence.rs

```diff
@@ -0,0 +1,37 @@
+use rusqlite::types::{FromSql, FromSqlError, FromSqlResult, ToSqlOutput, ValueRef};
+use rusqlite::{Result as RusqliteResult, ToSql};
+
+/// Classification of Pocket confidence keywords.
+#[derive(Clone, Copy, Debug, Eq, PartialEq, Hash)]
+#[repr(u8)]
+pub enum PocketKeywordConfidence {
```

Comment:
Suggestion: the module already has Pocket and Confidence, we can simply call it `KeywordConfidence` or `Confidence`.